creativecloud/features/interactive-components/slider-tray/slider-tray.css

```diff
@@ -359,6 +361,8 @@
     overflow: hidden;
     text-overflow: ellipsis;
     height: 64px;
+    word-break: normal !important;
```

Comment:
Can we make css selector more specific so that it gets prioritised in place of using important

Reply:
Even by making it more specific, the browser is adding styles on its own. So that is why I have added !important